assets/css/index.scss

```diff
@@ -27,3 +27,9 @@ img {
 code {
   font-size: 14px;
 }
+
+@media (max-width: 543px) {
+	.d-xs-none {
+		display: none!important;
```

Comment:
Would you mind adding a space before `!important`? We should probably setup SCSS linting, but thats for the future 🔜